_posts/2024-02-27-comet-donation.md

```diff
@@ -0,0 +1,106 @@
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+# Introduction
+The Apache Arrow PMC is pleased to announce the donation of the [Comet project],
+a native Spark SQL Accelerator built on [Apache Arrow DataFusion].
+
+Comet is an Apache Spark plugin that uses Apache Arrow DataFusion to
+accelerate Spark workloads. It is designed as a drop-in
+replacement for Spark's JVM based SQL execution engine and offers significant
+performance improvements for some workloads as shown below.
+
+```text
+   ┌─────────────────────────────────────────────────────────────────┐
+   │                                                                 │
+   │ ┌──────────┐ ┌────────────┐ ┌────────────┐       ┌────────────┐ │
+   │ │   SQL    │ │  Cluster   │ │  DAG/Task  │  ...  │  Executor  │ │
+   │ │ Planner  │ │  Manager   │ │ Scheduler  │       │            │ │
+   │ └──────────┘ └────────────┘ └────────────┘       └────────────┘ │
+   │                                                         │       │
+   └─────────────────────────────────────────────────────────────────┘
+     Spark (JVM Based)                                       │        
+                                  ┌ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─         
+                                                                      
+                                  │                                   
+                                  ▼                                   
+                 ┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓                   
+Comet Execution  ┃                                ┃                   
+Engine           ┃  ┌─────────────────────────┐   ┃                   
+(Native Code)    ┃  │ Apache Arrow DataFusion │   ┃                   
+                 ┃  └─────────────────────────┘   ┃                   
+                 ┃                                ┃                   
+                 ┃  ┌─────────────────────────┐   ┃                   
+                 ┃  │    Spark Compatible     │   ┃                   
+                 ┃  │  Expressions/Operators  │   ┃                   
+                 ┃  └─────────────────────────┘   ┃                   
+                 ┃                                ┃                   
+                 ┗━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┛                   
+```
+
+**Figure 1**: With Comet, users interact with the same Spark ecosystem, tools
+and APIs such as Spark SQL. Queries still run through Spark's mature and feature
+rich query optimizer and planner. However, the execution is delegated to Comet,
+which is significantly faster and more resource efficient than the  JVM based
+implementation.
```

Comment:
I assume you might already be planning to add more details, but I'd love to hear about any Comet-specific optimizations that were implemented. Similarly, I'd be interested to hear about DataFusion-specific features that made it a particularly good fit for Comet's implementation (DataFusion's focus on extensibility comes to mind).

Reply:
I defer to @sunchao  / @viirya  or one of the other comet authors for this. Maybe it would be a good topic to address in the initial release blog